Tests/FoundationInternationalizationTests/Formatting/Number/BinaryInteger+FormatStyleTests.swift

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+
+import XCTest
+
+import FoundationEssentials
+@testable import FoundationInternationalization
+
+#if canImport(BigInt) // Not included by default as it's a 3rd party library; requires https://github.com/attaswift/BigInt.git be added the package dependencies.  Proved useful in the past for finding bugs that only show up with large numbers.
+import BigInt
+#endif
```

Comment:
Instead of using `BigInt`, we should wait until `Decimal` is implemented and use it instead: https://github.com/apple/swift-foundation/issues/236

Reply:
To be clear, are you saying to remove this code entirely?  It's up to you, but FWIW I feel it's useful to keep accessible - e.g. while addressing feedback I locally re-enabled the BigInt package dependency in order to re-test things.